<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>OutputDialog</class>
	<widget class="QDialog" name="OutputDialog">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>1080</width>
				<height>591</height>
			</rect>
		</property>
		<property name="minimumSize">
			<size>
				<width>0</width>
				<height>0</height>
			</size>
		</property>
		<property name="maximumSize">
			<size>
				<width>1280</width>
				<height>720</height>
			</size>
		</property>
		<property name="font">
			<font>
				<family>Calibri</family>
				<weight>75</weight>
				<bold>true</bold>
			</font>
		</property>
		<property name="windowTitle">
			<string>Face Recognization System</string>
		</property>
		<property name="windowIcon">
			<iconset resource="resource.qrc">
				<normaloff>:/icon/icon.png</normaloff>:/icon/icon.png
			</iconset>
		</property>
		<widget class="QLabel" name="imgLabel">
			<property name="geometry">
				<rect>
					<x>10</x>
					<y>10</y>
					<width>640</width>
					<height>480</height>
				</rect>
			</property>
			<property name="minimumSize">
				<size>
					<width>4</width>
					<height>0</height>
				</size>
			</property>
			<property name="maximumSize">
				<size>
					<width>640</width>
					<height>480</height>
				</size>
			</property>
			<property name="text">
				<string/>
			</property>
		</widget>
		<widget class="QWidget" name="horizontalLayoutWidget">
			<property name="geometry">
				<rect>
					<x>10</x>
					<y>500</y>
					<width>641</width>
					<height>80</height>
				</rect>
			</property>
			<layout class="QHBoxLayout" name="horizontalLayout">
				<item>
					<widget class="QPushButton" name="RecognizeButton">
						<property name="maximumSize">
							<size>
								<width>400</width>
								<height>600</height>
							</size>
						</property>
						<property name="font">
							<font>
								<family>Roboto Condensed Light</family>
								<pointsize>30</pointsize>
								<weight>75</weight>
								<bold>true</bold>
							</font>
						</property>
						<property name="text">
							<string>Recognize</string>
						</property>
						<property name="checkable">
							<bool>true</bool>
						</property>
						<property name="autoRepeatInterval">
							<number>100</number>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QPushButton" name="RecordButton">
						<property name="maximumSize">
							<size>
								<width>400</width>
								<height>600</height>
							</size>
						</property>
						<property name="font">
							<font>
								<family>Roboto Condensed Light</family>
								<pointsize>30</pointsize>
								<weight>75</weight>
								<bold>true</bold>
							</font>
						</property>
						<property name="text">
							<string>Record</string>
						</property>
						<property name="checkable">
							<bool>true</bool>
						</property>
						<property name="autoRepeatInterval">
							<number>100</number>
						</property>
					</widget>
				</item>
			</layout>
		</widget>
		<widget class="QGroupBox" name="groupBox">
			<property name="geometry">
				<rect>
					<x>660</x>
					<y>40</y>
					<width>350</width>
					<height>291</height>
				</rect>
			</property>
			<widget class="QWidget" name="verticalLayoutWidget">
				<property name="geometry">
					<rect>
						<x>30</x>
						<y>30</y>
						<width>110</width>
						<height>211</height>
					</rect>
				</property>
				<layout class="QVBoxLayout" name="verticalLayout">
					<item>
						<widget class="QLabel" name="label_3">
							<property name="font">
								<font>
									<family>Roboto Condensed Light</family>
									<pointsize>12</pointsize>
									<weight>50</weight>
									<bold>false</bold>
								</font>
							</property>
							<property name="text">
								<string>Name : </string>
							</property>
						</widget>
					</item>
					<item>
						<widget class="QLabel" name="label_4">
							<property name="font">
								<font>
									<family>Roboto Condensed Light</family>
									<pointsize>12</pointsize>
									<weight>50</weight>
									<bold>false</bold>
								</font>
							</property>
							<property name="text">
								<string>Status : </string>
							</property>
						</widget>
					</item>
				</layout>
			</widget>
			<widget class="QWidget" name="verticalLayoutWidget_2">
				<property name="geometry">
					<rect>
						<x>120</x>
						<y>30</y>
						<width>200</width>
						<height>211</height>
					</rect>
				</property>
				<layout class="QVBoxLayout" name="verticalLayout_2">
					<item>
						<widget class="QLabel" name="NameLabel">
							<property name="font">
								<font>
									<family>Roboto Condensed Light</family>
									<pointsize>12</pointsize>
									<weight>50</weight>
									<bold>false</bold>
								</font>
							</property>
							<property name="text">
								<string/>
							</property>
						</widget>
					</item>
					<item>
						<widget class="QLabel" name="StatusLabel">
							<property name="font">
								<font>
									<family>Roboto Condensed Light</family>
									<pointsize>12</pointsize>
									<weight>50</weight>
									<bold>false</bold>
								</font>
							</property>
							<property name="text">
								<string/>
							</property>
						</widget>
					</item>
				</layout>
			</widget>
		</widget>
	</widget>
	<resources>
		<include location="resource.qrc"/>
	</resources>
	<connections/>
</ui>